src/main/java/az/edu/turing/service/impl/BookingServiceImpl.java

```diff
@@ -1,20 +1,102 @@
 package az.edu.turing.service.impl;
 
 import az.edu.turing.domain.dao.BookingDao;
+import az.edu.turing.domain.dao.FlightDao;
+import az.edu.turing.entities.BookingEntity;
+import az.edu.turing.entities.FlightEntity;
 import az.edu.turing.model.dto.BookingDto;
-import az.edu.turing.model.dto.request.CreateBookingRequest;
 import az.edu.turing.service.BookingService;
 
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
 public class BookingServiceImpl implements BookingService {
 
-    private final BookingDao bookingDao;
+    private BookingDao bookingDao;
+    private FlightDao flightDao;
 
-    public BookingServiceImpl(BookingDao bookingDao) {
+    public BookingServiceImpl() {
```

Comment:
...how does this constructor inject dependencies without dependencies?
please add `BookingDao bookingDao`, `FlightDao flightDao` in constructor